api/token/service.go

```diff
@@ -0,0 +1,91 @@
+package token
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+
+	"github.com/shellhub-io/shellhub/api/apierr"
+	"github.com/shellhub-io/shellhub/api/store"
+	"github.com/shellhub-io/shellhub/pkg/models"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type Service interface {
+	CreateToken(ctx context.Context, namespace string) (*models.Token, error)
+	GetToken(ctx context.Context, namespace string) (*models.Token, error)
+	DeleteToken(ctx context.Context, namespace string) error
+	SetReadOnly(ctx context.Context, name string) error
+}
+
+type service struct {
+	store store.Store
+}
+
+func NewService(store store.Store) Service {
+	return &service{store}
+}
+
+func (s *service) CreateToken(ctx context.Context, namespace string) (*models.Token, error) {
+	ns, err := s.store.GetNamespace(ctx, namespace)
+	if ns == nil {
```

Comment:
I'd prefer if you handle the error case first and then the `ns` error. So we follow the same pattern in the whole code.